### PR TITLE
Add client / conduit correspondences to readme

### PR DESCRIPTION
**Pull request by bermanjosh**
(Yeah, it's as ugly as we thought it'd be)

### CODE REVIEW

**Comment by bermanjosh** [comment]
#221

File: README.md [hunk 1]
```diff
@@ -8,3 +8,8 @@ and higher-level APIs for user convenience (http-conduit).
 
 If you're just getting started, it is recommended to use http-conduit, which
 [is documented in the Yesod book](http://www.yesodweb.com/book/http-conduit).
+
+Do to Haddock pulling the documentation directly from http-client, some "Since"
```
Comment (snoyberg) [inline README.md]: s/Do/Due

**Comment by bermanjosh** [comment]
@snoyberg Thanks - done

**Comment by snoyberg** [comment]
Thanks!

**Pull request merged**